model/classes/user/TaoAssetRoles.php

```diff
@@ -22,6 +22,9 @@
 
 namespace oat\taoMediaManager\model\classes\user;
 
+/**
+ * @deprecated Use \oat\taoMediaManager\model\user\TaoAssetRoles
+ */
 interface TaoAssetRoles
```

Comment:
Please, do not duplicated the constants. Instead extend the proper interface:

```suggestion
interface TaoAssetRoles extends \oat\taoMediaManager\model\user\TaoAssetRoles
```

And then remove the constants